app/views/application/robots.text.erb

```diff
@@ -52,10 +52,6 @@ Disallow: /works/*/transcription$
 <% else %>
 # Non-production, no robots please, although we let twitter in to test twitter integration
 
-# let twitter scrape cards metadata for testing
-User-agent: Twitterbot
-Disallow:
-
```

Comment:
I'd leave this in here just in case we still want to test twitter share on staging, it doesn't really hurt.